src/Global/molecules/UserMenu.tsx

```diff
@@ -186,6 +251,14 @@ const UserMenu: React.FC<IUserMenu> = ({
               Language / 言語
             </LanguageMenu>
         }
+        {(userDrawerFooter?.icon !== '' || userDrawerFooter?.title !== '' ) ?
```

Comment:
Please destructure these properties. This should allow you to simplify this kind of conditional also to simply:
`{(icon || title) ? ...`

Reply:
Done